analyzers/tests/SonarAnalyzer.UnitTest/Rules/AvoidExcessiveClassCouplingTest.cs

```diff
@@ -388,27 +370,28 @@ public A()
         var s2 = nameof(System.Action);
     }
 }
-",
-                new AvoidExcessiveClassCoupling { Threshold = 0 });
+").Verify();
 
 #if NET
         [TestMethod]
-        public void AvoidExcessiveClassCoupling_CSharp9() =>
-            OldVerifier.VerifyAnalyzerFromCSharp9Library(@"TestCases\AvoidExcessiveClassCoupling.CSharp9.cs", new AvoidExcessiveClassCoupling { Threshold = 1 });
+        public void AvoidExcessiveClassCoupling_CSharp9() => builderWithThreshold1
+            .AddPaths("AvoidExcessiveClassCoupling.CSharp9.cs")
```

Comment:
Line 375 above: The `#if` should be surrounded by empty lines before and after - it's a recent change in a coding style